0137-remove-cn470-from-iot-poc.md

```diff
@@ -15,45 +15,60 @@ This proposal removes the CN470 region from earning Proof-of-Coverage rewards on
 
-Currently the CN470 region accounts for 1,563DC ($0.01563) per Epoch of data transfer.
+Currently the CN470 region accounts for 1,563DC ($0.01563) per Epoch of data transfer. This figure is minute in proportion to the number of Hotspots deployed In China, compared to other countries of large Hotspot deployments
 
 ## Stakeholders
 
-All customers (e.g., sensor owners), hosts, and operators of the Helium IOT Network. Gateway manufacturers producing hotspots for the CN470 region.
+All customers (e.g., sensor owners), hosts, and operators of the Helium IOT Network. 
+- Gaming Hotspot owners and "Honest" Hotspot owners located in China are specifically negatively affected with 0 PoC earnings.
+- Hotspot owners in other regions are specifically positively affected with an average 15% boost in PoC earnings.
+- Hotspot manufacturers that are producing hotspots for the CN470 region.
 
 ## Detailed Explanation
 
 We propose to remove the CN470 region from PoC reward calculations.
 
-We then propose a new denylist focused on any Hotspots that have been asserted to the CN470 region.
+We then propose a new denylist focused on any Hotspots that have ever asserted to the CN470 region, now or in the past.  This includes Hotspots that were asserted in CN470 and have reasserted in other regions.
 
-This proposal does not remove Data Transfer capability for CN470 Hotspots.
+This proposal does not remove Data Transfer capability or rewards for CN470 Hotspots. 
 
 ## Implementation
 
-- Remove the CN470 region from PoC calculations by adding a check to the IOT PoC verifier to check if the witness is in the CN470 region and marking it invalid
-- Produce a new denylist for all hotspots that have ever been located in the cn470 region.
+- Remove the CN470 region from PoC calculations by adding a check to the IOT PoC verifier to check if the witness is in the CN470 region and marking it invalid.
+- Produce a new permanent denylist classifier for all hotspots that have ever been located in the CN470 region.
+  This classifier will be run weekly to identify new Hotspots asserting or reasserting in the CN470 region.
+  The denial of this hotspot cannot be removed by reassertion or changing the height or antenna properties.
+- Nova Labs/Heliium Foundation will deploy this HIP in their appropriate capacities.
+
+An implementation date is to be confirmed, but there is no requirement to delay implementation when ready to deploy.
 
 ## Alternatives
 
 - None at this time.
+- This HIP can be reversed by another HIP or IOT Working Group all-member Super Majority vote, if a new gaming identification classifier is identified as implementable that will be effective.
```

Comment:
Curious on the reason for the second option. Do we feel like the governance process won't address this quickly enough?

Reply:
I was ssuming that the IOTWG will act in the best interests of the network. But the community may not ever vote to give rewards away. The IOTWG could decide to use a HIP vote if they don't want the responsibility.

Reply:
Another HIP that provides explicit delegation to the WG feels better.